packages/sync-actions/src/order-actions.js

```diff
@@ -37,3 +38,56 @@ export function actionsMapDeliveries(diff, oldObj, newObj) {
 
   return handler(deliveriesDiff, oldObj.shippingInfo, newObj.shippingInfo)
 }
+
+export function actionsMapReturnsInfo(diff, oldObj, newObj) {
+  const returnInfoDiff = diff.returnInfo
+  if (!returnInfoDiff) return []
+
+  const handler = createBuildArrayActions('returnInfo', {
+    [CHANGE_ACTIONS]: (oldSReturnInfo, newReturnInfo) => {
+      const updateActions = Object.keys(returnInfoDiff).reduce(
+        (itemActions, key) => {
+          const { items = {} } = returnInfoDiff[key]
+          if (Object.keys(items).length > 0) {
+            return [
+              ...itemActions,
+              ...Object.keys(items).reduce((actions, index) => {
+                let itActions = []
```

Comment:
We can have both, payment and shipment changes at the same time, so we can't return inside every block as @daern91 pointed out in [this comment](https://github.com/commercetools/nodejs/pull/655#discussion_r200927490)

Reply:
interesting... then let's change approach...
the way you have your if-blocks now are sensitive to order. 
if you use push, then this can easily be re-factored later

```js

const x = []
if (shipment) x.push({})
if (payment) x.push({})

// instead of 

let x = []
if (shipment) x = [{}]
if (payment) x = [...x, {}]
```